app/src/main/res/values/strings.xml

```diff
@@ -76,6 +76,12 @@
     <!-- Content description for close button used in "contextual feature recommendation" (CFR) popups -->
     <string name="cfr_dismiss_button_default_content_description">Dismiss</string>
 
+    <!-- Total cookie protection "contextual feature recommendation" (CFR) -->
+    <!-- Text for the main message. -->
```

Comment:
Can we be more descriptive here? I feel like the localization team will come back to us if we just left the comment as it is.

Reply:
Changed to "Text for the message displayed in the CFR popup promoting the TCP feature".

Reply:
Maybe include `"contextual feature recommendation" (CFR)` and `total cookie protection (TCP)`

Reply:
Changed to " Text for the message displayed in the contextual feature recommendation popup promoting the total cookie protection feature. "